packages/odyssey-react-mui/src/PasswordField.tsx

```diff
@@ -128,14 +133,16 @@ const PasswordField = forwardRef<HTMLInputElement, PasswordFieldProps>(
           autoFocus={hasInitialFocus}
           data-se={testId}
           endAdornment={
-            <InputAdornment position="end">
-              <IconButton
-                aria-label="toggle password visibility"
-                onClick={togglePasswordVisibility}
-              >
-                {inputType === "password" ? <ShowIcon /> : <HideIcon />}
-              </IconButton>
-            </InputAdornment>
+            !isShowPasswordIconDisabled && (
+              <InputAdornment position="end">
+                <IconButton
+                  aria-label="toggle password visibility"
```

Comment:
@AnirudhMani-okta can you also add translating this string to the work for the Jira that covers translating the "show"/"hide" hardcoded strings?

Reply:
@shawyu-okta Addressed in [this PR](https://github.com/okta/odyssey/pull/1977). Is it preferred to merge the two into one PR?

Reply:
When you say "this string", are you referring to the `aria-label` or something else

Reply:
Oh yeah I meant the `aria-label`.

For some reason I thought the icons themselves had a string in them (like a `title` element) and thought those were the original target of the other PR for translations but looks like that's not the case and it's just this string on `aria-label` that needs to be handled for `PasswordField`

Reply:
No need to merge the PRs together imo

Reply:
Odyssey icons used to have a `title` or label or something on them. That was changed not long before I got on the project. I had a task to look into it, but then we added `IconButton` and labels in other components, so I didn't bother.

Calling it "eye open" hard-coded into the SVG isn't as descriptive as "show password".